src/row.rs

```diff
@@ -0,0 +1,113 @@
+/// A macro for creating a [`Row`] using vec! syntax.
+///
+/// `row!` is similar to the [`vec!`] macro, but it returns a [`Row`] instead of a `Vec`.
+///
+/// # Examples
+///
+/// * Create a [`Row`] containing a vector of [`Cell`]s:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::row;
+///
+/// let row = row!["hello", "world"];
+/// let row = row!["hello".red(), "world".red().bold()];
+/// ```
+///
+/// * Create a [`Row`] from a given [`Cell`] repeated some amount of times:
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::row;
+///
+/// let row = row!["hello"; 2];
+/// ```
+///
+/// * Use [`text!`], [`line!`] or [`span!`] macro inside [`row!`] macro.
+///
+/// ```rust
+/// # use ratatui::prelude::*;
+/// use ratatui_macros::{row, line, text, span};
+///
+/// let row = row![
+///     line!["hello", "world"], span!(Modifier::BOLD; "goodbye {}", "world"),
+///     text!["hello", "world"],
+/// ];
+/// ```
+///
+/// [`Row`]: crate::widgets::Row
+/// [`Cell`]: crate::widgets::Cell
+#[macro_export]
+macro_rules! row {
+    () => {
+        ratatui::widgets::Row::default()
+    };
+    ($cell:expr; $n:expr) => {
+        ratatui::widgets::Row::new(vec![ratatui::widgets::Cell::from($cell); $n])
+    };
+    ($($cell:expr),+ $(,)?) => {{
+        ratatui::widgets::Row::new(vec![
+        $(
+            ratatui::widgets::Cell::from($cell),
+        )+
+        ])
+    }};
+}
+
+#[cfg(test)]
+mod tests {
```

Comment:
A few useful missing tests simple to more complex:
- `row[]`
- `row[Cell::new("foo")]`
- `row[Cell::new("foo"); 2]`